src/components/dictionaryConcepts/components/helperFunction.js

```diff
@@ -62,3 +62,4 @@ export const MAP_TYPE = {
 export const TRADITIONAL_OCL_HOST = urlConfig.TRADITIONAL_OCL_HOST;
 
 export const CUSTOM_SOURCE = 'Custom';
+export const SELECTED_EVENT_NAME = 'source';
```

Comment:
Which kind or type of event is this?

Reply:
I have changed it to a more appropriate name.